src/components/IconButton/IconButton.tsx

```diff
@@ -58,6 +58,7 @@ export interface IconButtonProps extends VibeComponentProps {
    * Kind of button - like <Button />
    */
   kind?: ButtonType;
+  /** Control the button, set it in an active state */
```

Comment:
```suggestion
  /** Change the button appearance to active (pressed) state */
```

The control the button can be a bit confusing